src/marketplace/ListingsView.js

```diff
@@ -0,0 +1,146 @@
+//@flow strict
+import * as React from 'react';
+import {getTankComponent, verifyComponent} from '../armory/GetInventoryInfo.js';
+import Cookies from 'universal-cookie';
+import type {ComponentType} from '../armory/ComponentType.js';
+
+type Props = {|
+	//This is the type of item we are buying
+	sellerType: ComponentType,
+|};
+type State = {|
+	userId: string,
+|};
+
+//This allows for all the items that are for sale to be with in one array
+let itemsForSale = [{name: '', price: 0, amount: 0, sellerId: '', saleId: ''}];
```

Comment:
Please don't store variables outside the class. When you update this, the component doesn't rerender